File: btjs_page/btjs/mod/random/random_element.js



export function random_element(ARRAY) {
    // Verificamos si el array tiene 
    // elementos
    if (ARRAY.length === 0) {
      return null; // Si el array está 
      // vacío, retornamos null
    }
    // Generamos un índice aleatorio 
    // dentro del rango del array
    const indiceAleatorio = Math.floor(
        Math.random() * ARRAY.length);

    // Retornamos el elemento en el índice 
    // aleatorio
    return ARRAY[indiceAleatorio];
}

File: asp_2025/on_work/asp5/src/basic_things/core/basic/svc/random_element.py


from ..dependences import random

"""
    Funcion que elige un elemento aleatorio de un array
    y lo retorna.
"""
# return tipo del array
def random_element(array):
    if(not len(array) > 0):
        print("=>Error: void array from LogicFacade.random_element_from_array")
        return None
    # Obtener un índice aleatorio dentro del rango del array
    indice_aleatorio = random.randint(0, len(array) - 1)
    # Devolver el elemento del array en el índice aleatorio
    return array[indice_aleatorio]
